src/SFA.DAS.ApplyService.Domain/Entities/PersonInControl.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace SFA.DAS.ApplyService.Domain.Entities
+{ 
+
+public class PersonInControl
+    {
+        public string Name { get; set; }
+        public string DateOfBirth { get; set; }
```

Comment:
Rather than have a formatter do you think it'd be worth having a DateOfBirth value type?

Reply:
This conforms to the common output of MMM yyyy instead of d/m/y type of stuff, so it would still be a string.  Maybe the words MonthYearOfBirth would be a better name though?